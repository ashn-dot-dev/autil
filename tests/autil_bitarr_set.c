#define AUTIL_IMPLEMENTATION
#include "../autil.h"
#include "test.h"

int
main(void)
{
    struct autil_bitarr* const a = autil_bitarr_new(4096);

    // autil_bitarr_set is exercised by autil_bitarr_get.c
    // This just tests index out of bounds.
    autil_bitarr_set(a, 4096u, 1);

    puts("UNREACHABLE");
    autil_bitarr_del(a);
    return EXIT_SUCCESS;
}
