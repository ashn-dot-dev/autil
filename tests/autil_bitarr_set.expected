error: [autil_bitarr_set] Index out of bounds (4096)
